src/components/HeaderBar.vue

```diff
@@ -70,21 +69,23 @@ const props = defineProps({ allSports: { type: Array, required: true } })
 }
 
 button{
-  height: 64px;
+  padding: 16px;
   transition: background-color ease-in-out .3s;
   background-color: #101c33;
-  color: #a7a7a7;
+  transition: opacity ease-in-out .2s;
+  color: #ffff;
+  opacity: 64%;
   cursor: pointer;
```

Comment:
Using opacity is a good solution, but since we don't want the _entire_ element to become opaque we want to target just the text and the icon (both of which use color to get their color).

The text is easy (using `rgba`) but for the icons look up svg `currentColor`. It might require you to change your icons, btw.